docs/parallel_training/overview.rst

```diff
@@ -0,0 +1,53 @@
+Features
+========
+
+Highlights
+##########
+
+Simple usage
+^^^^^^^^^^^^
+| With a single `@parl.remote_class` decorator, we can access cluster's remote
```

Comment:
With a single `@parl.remote_class` decorator, users can implement parallel training easily, and  not having to care about stuff of multi-processes, network communication.